### PR TITLE
fix: Deploy bucket in configured location

### PR DESCRIPTION
**Pull request by Gnucki**
This is a basic correction for the issue https://github.com/serverless/serverless-google-cloudfunctions/issues/226

Tell me if I can do something better !

Note that this PR is similar to https://github.com/serverless/serverless-google-cloudfunctions/pull/220

### CODE REVIEW

File: package/lib/prepareDeployment.js [hunk 1]
```diff
@@ -30,9 +31,13 @@ module.exports = {
   },
 };
 
-const updateBucketName = (bucket, name) => {
+const updateBucket = (bucket, name, location) => {
   const newBucket = _.cloneDeep(bucket);
   newBucket.name = name;
+  if (location) {
+    newBucket.properties = newBucket.properties || {};
```
Comment (medikoo) [inline package/lib/prepareDeployment.js:38]: Let's write it as `if (!newBucket.properties) newBucket.properties = {}`

Comment (Gnucki) [reply package/lib/prepareDeployment.js]: Improvement done.

Of course, I tested it ;)

**Comment by curtismenmuir** [comment]
When will the next release be available with this fix?

**Comment by medikoo** [comment]
@curtismenmuir just published v3.1.1

**Comment by curtismenmuir** [comment]
@medikoo Perfect, thank you :)

**Pull request merged**